jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/StartEventTest.java

```diff
@@ -347,10 +345,10 @@ public void beforeProcessStarted(ProcessStartedEvent event) {
                     list.add(event.getProcessInstance().getId());
                 }
             });
-            assertEquals(0, list.size());
+            Assertions.assertThat(list.size()).isEqualTo(0);
             // Timer in the process takes 500ms, so after 2.5 seconds, there should be 5 process IDs in the list.
             countDownListener.waitTillCompleted();
-            assertEquals(5, getNumberOfProcessInstances("MultipleStartEvents"));
+            Assertions.assertThat(getNumberOfProcessInstances("MultipleStartEvents")).isEqualTo(2);
```

Comment:
In this test you changed a number of processes from 5 to 2, so please update the comment accordingly.